inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/service/RecommendationServiceImpl.java

```diff
@@ -1009,6 +1009,7 @@ public Predictions computePredictions(User aUser, Project aProject,
 
                         predictions.putPredictions(layer.getId(), suggestions);
                     }
+                    // Catching Throwable is intentional here as we want to continue the execution even if a particular recommender fails.
```

Comment:
These lines are very long. Please install checkstyle (see [instructions](https://inception-project.github.io//releases/0.12.4/docs/developer-guide.html#_setting_up_checkstyle_and_formatting)) to find style violations in your code.